Riot/Modules/Settings/Notifications/ViewModel/NotificationSettingsViewModelType.swift

```diff
@@ -0,0 +1,29 @@
+// File created from ScreenTemplate
+// $ createScreen.sh Settings/Notifications NotificationSettings
+/*
+ Copyright 2021 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+
+protocol NotificationSettingsViewModelCoordinatorDelegate: AnyObject {
+    func notificationSettingsViewModel(_ viewModel: NotificationSettingsViewModelType, didCompleteWithUserDisplayName userDisplayName: String?)
```

Comment:
`didCompleteWithUserDisplayName` is in a few of the delegates, possibly left over from the template?